src/components/grid-aware/Modal/Modal.js

```diff
@@ -13,10 +14,15 @@ const Modal = ({
   contentLabel,
   noBezel,
 }) => {
-  const modalClassName = `${s.content} ${noBezel ? s.noBezel : s.withBezel}`;
-  const buttonClassName = `${s.closeButton} ${
-    noBezel ? s.noBezel : s.withBezel
-  }`;
+  const modalClassName = classNames(s.content, {
+    [s.noBezel]: noBezel,
+    [s.withBezel]: !noBezel,
+  });
```

Comment:
Similar to slightly different from my other comment about the light/dark theme, I think this you may be able to do by passing in just a single value for the noBezel/withBezel thing, but where you conditionally select the appropriate class name beforehand. Could you try the following?

```suggestion
  const modalClassName = classNames(s.content, noBezel ? s.noBezel : s.withBezel);
```

If you feel it's more readable, you can also take that ternary expression and assign it to a new variable just above this line.